.github/workflows/test.yml

```diff
@@ -1,32 +1,45 @@
 name: Test
 
 on:
-  - pull_request
+  pull_request: {}
+  push:
+    branches:
+      - master
+
+env:
+  BUNDLE_WITHOUT: release
 
 jobs:
-  test:
+  rspec:
```

Comment:
I'd be tempted to name this `spec`, after `test:spec`

Reply:
done